website/docs/pubsub.md

```diff
@@ -125,6 +126,9 @@ Dapr: https://dapr.io/
           - name: go-sub
             image: fake-subscriber:latest
             imagePullPolicy: Never
+            env:
+            - name: AUDIT_CHANNEL
```

Comment:
what is consuming this env var? it looks like it got removed here: https://github.com/open-policy-agent/gatekeeper/pull/3339/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L228